### ISSUE TITLE
Player name and coordinates overlap

### ISSUE THREAD

**Issue by braoult** [mliebelt/pgn-viewer#452]
White player name and coordinates overlap.

To reproduce (Configuration Builder):
Load `Evergreen (No board)`,  tick `Show Coordinates` and untick `Coords Inner`, to get coordinates outside the board.
We get:

![image](https://github.com/mliebelt/PgnViewerJS/assets/8958429/00d4b308-36e8-4284-91b6-bcba0e639730)

Now, tick `Headers`, to add players name. We get :

![image](https://github.com/mliebelt/PgnViewerJS/assets/8958429/7dfc7675-a871-44c5-8d1b-f9f516184b55)

Interesting part is that **the buttons bar is moved down**, as if the space used by player name was taken into account, and only name placement is actually wrong.

**Comment by mliebelt** [comment]
True. There is a correction done, but seems to be on the wrong element. I will check that, by producing an example that contains both elements (headers and coordinates out) to reproduce it. Should not be too complicated .... ( I hope :-) ).

**Comment by mliebelt** [comment]
I think I found the reason, does not want to do a release because of that.

**Comment by braoult** [comment]
No problem. For my information, is it a CSS-only problem, that I could try locally ?

**Comment by mliebelt** [comment]
No. There is some logic needed to set the CSS dependent on the configuration. So not so easy to fix in CSS only ...
